src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs

```diff
@@ -266,5 +266,28 @@ public Task GetReplicationReconnectionQueue()
             }
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/studio-tasks/resolveMerge", "GET", "/databases/{databaseName:string}/studio-tasks/resolveMerge?docId={documentId:string}")]
```

Comment:
Need a better name, `suggest-conflict-resolution` ? We aren't resolving anything here.